apps/ledger-live-desktop/src/renderer/screens/exchange/index.jsx

```diff
@@ -58,6 +60,7 @@ const DEFAULT_MULTIBUY_APP_ID = "multibuy";
 // Exchange (Buy / Sell) as a live app screen
 const LiveAppExchange = ({ appId }: { appId: string }) => {
   const { state: urlParams } = useLocation();
+  const locale = useSelector(languageSelector) || "en";
```

Comment:
```suggestion
  const locale = useSelector(languageSelector);
```

No need to provide a fallback value, the selector already handles this logic 👍 

cf. https://github.com/LedgerHQ/ledger-live/blob/615114bd72df24265061cffa24d04e60eaf91914/apps/ledger-live-desktop/src/renderer/reducers/settings.js#L415